Promise.playground/Pages/Basic Usage.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,36 @@
+
+import Promise  // <-- If there is an error here, build the project first.
+
+/*
+
+ Basic usage of a Promise.
+
+ */
+
+/// Helper function to build/fulfill a Promise.
+/// - returns: A Promise that fulfills with the given string, after a delay.
+func promisedString(str: String) -> Promise<String> {
+    return Promise<String>(work: { fulfill, reject in
+        print("sleeping…")
+        sleep(1)
+        print("done sleeping")
+        fulfill(str)
+    })
+}
+
+
+promisedString("simple example").then { result in
+    print("Got result: ", result)
+}
```

Comment:
i know i haven't published a style guide yet, but i'm not using trailing block syntax for this project. i'll comment each time it needs to be fixed